### PR TITLE
[#112] Authorization

### PR DESCRIPTION
**Pull request by vlad-tokarev**
* Add helm chart for openpolicyagent deployment
* Write MVP version of authorization policies (policy.rego file)

### CODE REVIEW

File: helms/odahu-flow-authorization/files/openpolicyagent/policy.rego [hunk 1]
```diff
@@ -0,0 +1,65 @@
+package odahu
```
Comment (mcdoker18) [inline helms/odahu-flow-authorization/files/openpolicyagent/policy.rego]: IMHO, this is the wrong place for a policy file.
I thought that we would keep policies in the odahu-automation or legion-profiles repository.
I don't want to change the helm chart whenever my policies are changed.

File: helms/odahu-flow-authorization/values.yaml [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+# Declare variables to be passed into your templates.
+
+# enabled is bool trigger to activate/deactivate chart resources
+enabled: true
+
+replicaCount: 1
+
+image:
+  repository: openpolicyagent/opa
+  tag: latest-istio
```
Comment (mcdoker18) [inline helms/odahu-flow-authorization/values.yaml]: Pin a some version

File: helms/odahu-flow-authorization/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+name: odahu-flow-authorization
```
Comment (mcdoker18) [inline helms/odahu-flow-authorization/Chart.yaml]: Why not `odahu-flow-opa`?

File: helms/odahu-flow-authorization/values.yaml [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
```
Comment (mcdoker18) [inline helms/odahu-flow-authorization/values.yaml]: A user should override the policies.

File: helms/odahu-flow-authorization/files/openpolicyagent/config.yaml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+decision_logs:
```
Comment (mcdoker18) [inline helms/odahu-flow-authorization/files/openpolicyagent/config.yaml]: Do you use this file anywhere?

**Review by mcdoker18 — CHANGES_REQUESTED** [review]

File: helms/odahu-flow-opa/files/openpolicyagent/policy.rego [hunk 6]
```diff
@@ -0,0 +1,63 @@
+package odahu
+
```
Comment (mcdoker18) [inline helms/odahu-flow-opa/files/openpolicyagent/policy.rego]: Did you not forget to delete this file?

**Review by mcdoker18 — CHANGES_REQUESTED** [review]

**Review by mcdoker18 — APPROVED** [review]

**Review by maxkochubey — APPROVED** [review]

**Pull request merged**